server/index.js

```diff
@@ -1,25 +1,28 @@
-/* eslint-disable no-console */
 const path = require('path');
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
-const Restaurant = require('../database/Restaurant.js');
+const cors = require('cors');
+const db = require('../database/index');
```

Comment:
Doesn't seem like this line is required by any other lines?

Reply:
Removed line 6